activities/TamTamMicro.activity/index.html

```diff
@@ -11,10 +11,12 @@
 	href="lib/sugar-web/graphics/css/sugar-200dpi.css">
 <link rel="stylesheet" href="css/activity.css">
 <link href="lib/enyo/enyo.css" rel="stylesheet" type="text/css" />
+<script src="https://unpkg.com/tone@next/build/Tone.js"></script>
```

Comment:
Do not reference library on the web. Everything should be copied locally to allow offline mode

Reply:
4f99ebb15387e3e726bf0df0da8199b0d339e31f